src/spaceone/cost_analysis/service/job_service.py

```diff
@@ -57,19 +57,25 @@ def create_jobs_by_data_source(self, params):
         """Create jobs by domain
 
         Args:
-            params (dict): {}
+            params (dict): {
+                "sync_hour": int
+            }
 
         Returns:
             None
         """
 
         for data_source_vo in self._get_all_data_sources():
```

Comment:
I think at `_get_all_data_sources()`method can filter enabled schedule state data sources.

```suggestion
        for data_source_vo in self._get_all_data_sources():
```